cluster/images/kubemark/BUILD

```diff
@@ -0,0 +1,34 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load("@io_bazel_rules_docker//docker:docker.bzl", "docker_build", "docker_push")
+
+docker_build(
+    name = "image",
+    base = "@debian//image",
+    entrypoint = ["/kubemark"],
+    files = [":kubemark"],
```

Comment:
this should depend on the kubemark binary properly, unlike the way the Dockerfile/Makefile is currently written.

Reply:
you mean have another bazel rule to build the kubemark binary?

Reply:
well, the rule should already exist. you probably want `//cmd/kubemark` here.